source/_integrations/zha.markdown

```diff
@@ -402,6 +402,16 @@ Note that not all devices support binding as it depends on the Zigbee implementa
 
 Binding a remote directly to a bulb or group has the benefit of faster response time and smoother control. This greatly improves user feedback experience functions like dimming as the remote then directly dims the lightbulb and thus does not have to make the software roundtrip via the ZHA coordinator.
 
+## Zigbee backup and restore in ZHA
+
+Zigbee Home Automation (ZHA) integration now features Zigbee network backup, restore/recovery, and migrating between Zigbee coordinators since Home Assistant 2022.9 release. Backups are taken automatically however a single backup to file for easy download can also be manually created from the configuration page under Network Settings.
+
+After restoring a Home Assistant backup, you can re-configure ZHA and migrate to a new Zigbee Coordinator adapter without any loss of your settings or devices that were connected. This is helpful if your current radio fails or a new radio adapter type and model comes out that you may want to migrate to.
+
+The ZHA integration currently store and export the Zigbee network backup files in a [zigpy](https://github.com/zigpy/zigpy) specific format.
```

Comment:
I'm not sure how this is relevant for the end user? I suggest removing it.

Reply:
@puddly made it sound relevant because it imply that the exported file can only be imported as-is into ZHA and other Zigbee implementations that support the zigpy specific format, and are other Zigbee implementation out these that may or may not support import these backup files too for migration, if not now then maybe in the future.

Reply:
There is nothing supporting it. Even if there is something that supported it, let's call those out in the future instead of names of underlying libraries that an average end-user shouldn't care about in their daily use.

Please remove it.

Reply:
Yeah, I don't think it's relevant. The more irrelevant documentation, the harder it is for someone to actually find useful information.

It's not a reasonable assumption for one to assume that a backup is compatible across software. Incompatibility is always the default.

Reply:
OK I have now removed the line. about zigpy specific format.